osquery/core/watcher.cpp

```diff
@@ -48,10 +48,20 @@ struct LimitDefinition {
 };
 
 struct PerformanceChange {
+  /// Number of consecutive times the CPU limit was exceeded.
   size_t sustained_latency{0};
+  /// Memory footprint.
   uint64_t footprint{0};
-  uint64_t iv{0};
```

Comment:
This is just a rename, what does `iv` mean? From a comment it seems it was the check interval.